onnx/defs/training/defs.cc

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) ONNX Project Contributors.
+// Licensed under the MIT license.
+
+#include <algorithm>
+#include <cmath>
+#include "onnx/defs/function.h"
+#include "onnx/defs/schema.h"
+
+namespace ONNX_NAMESPACE {
+static const char* Gradient_ver12_doc = R"DOC(
+Gradient operator computes the derivatives of some tensors with respect to
```

Comment:
change to "the partial derivatives of a specified tensor with respect to some other tensors"? I think of "dy/dx" as the derivative of y with respect to x.

Reply:
No problem.